rke/k8s_rke_system_images.go

```diff
@@ -6438,6 +6438,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.14
```

Comment:
v2.5.15 will be released shortly (likely this week, there is already RC out - https://github.com/rancher/rancher/releases/tag/v2.5.15-rc1), but we won't be releasing KDM with it, so let's change it to:
```suggestion
		// Enabled out of band post v2.5.15
```